### PR TITLE
added Linux section and a great video

### PR DESCRIPTION
**Pull request by ashutoshsao**
added a video that is very intuitive and shows the history of linux in brief

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,6 +271,11 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
   
 ---
 
+## Linux
```
Comment (Ashish-khanagwal) [inline README.md:274]: Hey, Add this section above Machine learning section in alphabetical order

**Review by Ashish-khanagwal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ashutoshsao) [reply README.md:274]: sure

Comment (ashutoshsao) [reply README.md]: done !

**Comment by ashutoshsao** [comment]
can you check if its ok now

**Review by Ashish-khanagwal — APPROVED** [review]
Looks good to me

Thank you for contributing and keep going

**Comment by ashutoshsao** [comment]
your welcome !

**Comment by Ashish-khanagwal** [comment]
Don't forget to join discord server of the community from the home page of repo🔥

**Pull request merged**